pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -373,6 +373,14 @@ def ddp_train(self, process_idx, model):
             model, optimizers = model.configure_apex(amp, model, self.optimizers, self.amp_level)
             self.optimizers = optimizers
 
+            # Reinitialize optimizer.step properties added by schedulers
+            for lr_scheduler in self.lr_schedulers:
```

Comment:
maybe we can run this only if `trainer.use_amp`

Reply:
this is evaluated inside the `if` statement:
`if self.use_amp and not self.use_native_amp:`
so it should already be taken care of

Reply:
`use_native_amp` does not need apex...